graphql/schema.graphql

```diff
@@ -86,6 +86,8 @@ type Mutation {
   setTaskPriority(taskId: String!, priority: Int!): Task!
   restartTask(taskId: String!): Task!
   saveSubscription(subscription: SubscriptionInput!): Boolean!
+  moveIssueToSuspectedIssue(annotationId: String!, apiIssue: AnnotationIssue!): Boolean!
```

Comment:
sometimes we display success or error banners in our UI based on the outcome of a mutation. any additional information about the `issue` you think is important to show in the banner can be added to the return type of the mutation.

Reply:
This probably shouldn't even have a banner, the issue should just be moved, so I think this will be enough 👍